libs/vue/src/schematics/application/schematic.ts

```diff
@@ -108,9 +108,10 @@ function getEslintConfig(options: NormalizedSchema) {
   const eslintConfig: any = {
     extends: [
       `plugin:vue/${options.isVue3 ? 'vue3-' : ''}essential`,
+      'eslint:recommended',
       '@vue/typescript/recommended',
-      'prettier',
-      'prettier/@typescript-eslint',
+      '@vue/prettier',
+      '@vue/prettier/@typescript-eslint',
```

Comment:
This shouldn't be changed. It configures Prettier to be ran as lint rules, but Nx uses Prettier separately from ESLint.